custom_components/tuya_local/devices/poolsana_heatpump.yaml

```diff
@@ -0,0 +1,113 @@
+name: PoolSana InverPower Next
```

Comment:
A `products` section with the product id and name (branding) would be useful, and the above name should then be changed to be generic ("Pool heat pump") in case other brands match the same device config.

Product id can be obtained from iot.tuya.com info, or `python -m tinytuya scan`

Reply:
Sure, I've added/corrected this.